src/Components/WhatdowedoComponent/WhatDoWeDo.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import Text from "../../elements/text/Text";
+import Container from "../../elements/wrapper/container/Container";
+import Row from "../../elements/wrapper/row/Row";
+import CardV1 from "../cardv1/CardV1";
+import styles from "./WhatDoWeDo.module.scss";
+
+const WhatDoWeDo = () => {
+  return (
+    <Container className={`${styles.margin}`}>
+      <Row className={`${styles.row}`}>
```

Comment:
if there is no other variables or strings included inside className, no need to use string interpolation className={styles.row}

Reply:
noted